hasura/metadata/cron_triggers.yaml

```diff
@@ -1,21 +1,21 @@
-- name: backfillReputation
-  webhook: '{{HASURA_API_BASE_URL}}/cron/backfillReputation'
+- name: backfillFarcasterConnect
+  webhook: '{{HASURA_API_BASE_URL}}/cron/backfillFarcasterConnect'
   schedule: '* * * * *'
   include_in_metadata: true
   payload: {}
   headers:
     - name: verification_key
       value_from_env: HASURA_EVENT_SECRET
-  comment: make sure everyone has a reputation, in batches
-- name: backfillFarcasterConnect
-  webhook: '{{HASURA_API_BASE_URL}}/cron/backfillFarcasterConnect'
+  comment: auto connect farcaster accounts for those with matching address
+- name: backfillReputation
+  webhook: '{{HASURA_API_BASE_URL}}/cron/backfillReputation'
   schedule: '* * * * *'
   include_in_metadata: true
   payload: {}
   headers:
     - name: verification_key
```

Comment:
this is jsut diff noise i think. maybe worth committing if we think its going to stay stable... but if not then best to leave out.